api/aperture/policy/language/v1/policy.proto

```diff
@@ -357,4 +358,6 @@ message NestedCircuit {
   string name = 4;
   // Short description of the nested circuit component. This description is displayed by graph visualization tools.
   string short_description = 5;
+  // Arbitrary configuration for the nested circuit component. This configuration is displayed by graph visualization tools.
+  google.protobuf.Any config = 6;
```

Comment:
This will show up in the spec now, right? Should we just provide this as a string or something?

Reply:
Yes, do you see a problem with that? I don’t really need to do this through the spec, can set the config in code instead

Reply:
Name and description fields were already there so added this. But we can very much set these fields in code where required instead of relying on the spec

Reply:
Pushed a new commit, taking code-based approach now. Removed all visualization specific fields from the policy spec.